### PR TITLE
Update calculator conundrum to use test runner v3 features

### PR DESCRIPTION
**Pull request by aldotele**
# pull request

this PR addresses the calculator-conundrum task on issue #2356 

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/java/blob/main/POLICIES.md#event-checklist)

### CODE REVIEW

File: exercises/concept/calculator-conundrum/src/test/java/CalculatorConundrumTest.java [hunk 1]
```diff
@@ -53,16 +60,30 @@ public void throwExceptionForUnknownOperation() {
     }
 
     @Test
+    @Tag("task:2")
+    @DisplayName("The calculate method throws an IllegalOperationException with custom message when passing null as operation")
```
Comment (sanderploegsma) [inline exercises/concept/calculator-conundrum/src/test/java/CalculatorConundrumTest.java:64]:
```suggestion
    @DisplayName("The calculate method throws an IllegalArgumentException with custom message when passing null as operation")
```

File: exercises/concept/calculator-conundrum/src/test/java/CalculatorConundrumTest.java [hunk 2]
```diff
@@ -53,16 +60,30 @@ public void throwExceptionForUnknownOperation() {
...
     public void throwExceptionForNullAsOperation() {
         assertThatExceptionOfType(IllegalArgumentException.class)
                 .isThrownBy(() -> new CalculatorConundrum().calculate(66, 65, null))
                 .withMessage("Operation cannot be null");
     }
 
     @Test
+    @Tag("task:2")
+    @DisplayName("The calculate method throws an IllegalOperationException with custom message when passing an empty operation")
```
Comment (sanderploegsma) [inline exercises/concept/calculator-conundrum/src/test/java/CalculatorConundrumTest.java:73]:
```suggestion
    @DisplayName("The calculate method throws an IllegalArgumentException with custom message when passing an empty operation")
```

File: exercises/concept/calculator-conundrum/src/test/java/CalculatorConundrumTest.java (hunk 1 above)
Comment (aldotele) [reply exercises/concept/calculator-conundrum/src/test/java/CalculatorConundrumTest.java]: thanks for the review, I just updated with a fixing commit

**Review by sanderploegsma — APPROVED** [review]
LGTM 👍

**Pull request merged**